README.md

```diff
@@ -1,2 +1,43 @@
-# Morse-Code
-Help us translate the secret message inside the bottle.
```

Comment:
- [ ] I think it would be better if you add the project name and description in your README to showcase your project so kindly add it 🚀💯🔥🔥

Reply:
Hello! Thanks for the observation. Apparently, we had an accident putting together the readme file and deleted the project name. We have already put it back on. 😃

Reply:
@Reem-lab 
I've also added some examples to the main.rb file below to make it easier to identify how to test the code. Thank you very much for your comments!